scripts/src/java/org/oppia/android/scripts/common/RepositoryFileTest.kt

```diff
@@ -0,0 +1,126 @@
+package org.oppia.android.scripts.common
+
+import com.google.common.truth.Truth.assertThat
+import org.junit.Before
+import org.junit.Rule
+import org.junit.Test
+import org.junit.rules.TemporaryFolder
+import java.io.File
+
+/** Tests for [RepositoryFile]. */
+class RepositoryFileTest {
+  @Rule
+  @JvmField
+  var tempFolder = TemporaryFolder()
+
+  @Before
+  fun setUpTests() {
```

Comment:
Just ``setUp`` is fine (and conventional with what we name this elsewhere).

Reply:
Done